src/components/Layout/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+
+import Container from "../Container/Container";
+import UserSettings from "./UserSettings";
+import Searchbar from "../Searchbar/Searchbar";
+
+const Header = () => {
+  return (
+    <Container as="header" className="flex justify-between p-5">
+      <Link href="/" className="font-logo text-3xl leading-6">
+        Next{<br />}Fakeshop
```

Comment:
Редко я конечно такое с `br` вижу, но наверное сойдёт? Как альтернатива, это можно обернуть в два блока с `display: block`, но это уже скорее из рода другого подхода. Выйдет более громоздко конечно...
```suggestion
        <div className="block">Next</div>
        <div className="block">Fakeshop</div>
```

Reply:
Да у меня символ переноса строки не работал и мне было лень разбираться. Пока так сойдет